app/functoria_app.ml

```diff
@@ -127,13 +125,13 @@ let pp_packages fmt l =
          (fun fmt x -> pf fmt "%S, \"%%{%s:version}%%\"" x x)
         ) l
 
-let pp_dump_info module_name fmt i =
+let pp_dump_pkgs module_name fmt (name, pkg, libs) =
   Fmt.pf fmt
     "%s.{@ name = %S;@ \
      @[<v 2>packages = %a@]@ ;@ @[<v 2>libraries = %a@]@ }"
-    module_name (Info.name i)
-    pp_packages (Info.package_names i)
-    pp_libraries (Info.libraries i)
+    module_name name
+    pp_packages (List.sort String.compare (String.Set.elements pkg))
+    pp_libraries (List.sort String.compare (String.Set.elements libs))
```

Comment:
According to the doc, `Set.elements` is guaranteed to be sorted. You could either remove the sorting, or remove the transformation to a set.

Reply:
fixed in 520079e (removed `List.sort String.compare`)